examples/samples/topicsGettingStarted.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  generateUuid,
+  Namespace,
+  SendableMessageInfo,
+  SubscriptionClient,
+  TopicClient
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscription1 = process.env.SUBSCRIPTION_NAME1 || "";
+const subscription2 = process.env.SUBSCRIPTION_NAME2 || "";
+const subscription3 = process.env.SUBSCRIPTION_NAME3 || "";
+
+console.log("str: ", str);
+console.log("path: ", topic);
```

Comment:
I think we should log the 3 subscriptions as well.

Reply:
Ok!